README.md

```diff
@@ -222,6 +222,8 @@ Converters from various static schedule formats to GTFS.
 - [GTFS-realtime Nagio Plugin](https://github.com/OneBusAway/onebusaway-gtfs-realtime-nagios-plugin) - Provides a [Nagios](https://www.nagios.org/) plugin for monitoring a GTFS-realtime feed
 - [GTFS-realtime-test-service](https://github.com/CUTR-at-USF/gtfs-realtime-test-service) - A tool for mocking GTFS-realtime feed content (e.g., for use in testing a GTFS-realtime consuming application)
 
+#### GTFS Realtime Documentation
+- [GTFS-realtime Autodoc](https://laidig.github.io/gtfs-rt-autodoc/index.html) - Automatically generated documentation for GTFS-realtime, including some extensions. 
```

Comment:
Can you please move this to the top of the GTFS-RT section to mirror the GTFS section, and add link to the canonical markdown docs as well?